### PR TITLE
Add Lineage configuration.

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description

Add Lineage configuration to enable automatic PRs from upstream. 

:eyes: Please verify that the correct upstream repo was selected.

### CODE REVIEW

**Comment by jsf9k** [comment]
I manually merged in the upstream changes in order to get the GitHub Actions to pass.

**Review by jsf9k — APPROVED** [review]
Nice work @jsf9k!  You're the best!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for review when someone
 # opens a pull request.
-*       @dav3r @felddy @jsf9k @mcdonnnj @kcoble @cisagov/team-ois
+*       @dav3r @felddy @hillaryj @jsf9k @mcdonnnj @cisagov/team-ois
```
Comment (mcdonnnj) [inline .github/CODEOWNERS:7]: Should we leave Krysta since she's the author?

**Review by mcdonnnj — APPROVED** [review]
The great @jsf9k strikes again! 💪💪💪I had one question about the code owners for this repo.

File: .github/CODEOWNERS (hunk 1 above)
Comment (jsf9k) [reply .github/CODEOWNERS:7]: I puzzled over that too.  @kcoble doesn't work here anymore, and I doubt she wants to maintain this repo.

If I'm wrong, @kcoble, just let me know and I'll happily add you again!

Comment (felddy) [reply .github/CODEOWNERS:7]: I miss @kcoble .  We had some fun conversations in the lab!

**Pull request merged**